wc-smooth-generator.php

```diff
@@ -8,8 +8,9 @@
  * Author URI: https://woocommerce.com
  *
  * Tested up to: 5.7
+ * Requires PHP: 7.4
```

Comment:
Should we update this to 8.0.2? So far if I try and install it on any site with less than that version of PHP (such as 7.4 and 8.0.0) I get the error that I need to be running at least 8.0.2 on the site.

Reply:
@zhongruige Thanks for catching this. Updated.